src/features/TodoList.js

```diff
@@ -0,0 +1,22 @@
+import React,{useState} from 'react'
+import TodoGenerator from './TodoGenerator'
+import TodoGroup from './TodoGroup'
+
+export default function TodoList() {
+    const initArray = [];
+    const [size,setSize] = useState(0);
+    const [textList, setText] = useState(initArray);
+
+    const addSize = () => {
+        setSize(size + 1);
+    }
+    const editText = (text) => {
+        textList.push(text);
+    }
+    return (
+        <div class= "todoList">

```

Comment:
should be className ,
![image](https://user-images.githubusercontent.com/56144156/207204190-b77dba0e-bca4-4646-8a55-19152382802b.png)